test/browser/video-sharing.js

```diff
@@ -55,6 +58,16 @@ describe('Video Sharing', async function() {
         ]);
     });
 
+    it('function: findCall, verifies stream objects of users', async () => {
+        await sleep(5000); // wait for all user's video streams to update
+        const res = await Promise.all([
+            client.findCall(call.callId),
+            peerUser1.exec('findCall', call.callId),
+            peerUser2.exec('findCall', call.callId)
+        ]);
+        assert(res.every(userCall => userCall.localStreams.video && userCall.participants.every(participant => participant.streams.video)));
```

Comment:
rename userCall to call.

Not sure what you are testing here. 
- client.findCall gets the local call for the bot, but the bot doesn't share video.
- why are you joining with 2 peer users?



More importantly you are not removing the Urls which cause the issues. E.g. see line 56